tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Steps/EmailSteps.cs

```diff
@@ -37,46 +44,116 @@ public async Task EmailIsNotSent()
             emails.Should().BeNullOrEmpty();
         }
 
-        [Then(@"the email sent contains the following information")]
-        public async Task ThenEmailContains(Table table)
+        [Then(@"the email contains the following information")]
+        public async Task ThenTheEmailContainsTheFollowingInformation(Table table)
         {
             var expectedContents = table.CreateSet<EmailContentsTable>().First();
 
             var expected = new
             {
-                Attachments = new List<EmailAttachmentData>
-                {
-                    new EmailAttachmentData(
-                        Encoding.UTF8.GetBytes($"{expectedContents.AttachmentHeader1}\r\n{expectedContents.OrderId}\r\n"),
-                        expectedContents.FileName,
-                        new ContentType("text/csv"))
-                },
-                From = new List<EmailAddress>
-                {
-                    new EmailAddress(Empty, expectedContents.From)
-                },
+                From = new List<EmailAddress> { new EmailAddress(Empty, expectedContents.From) },
+                To = new List<EmailAddress> { new EmailAddress(Empty, expectedContents.To) },
                 expectedContents.Text,
                 expectedContents.Subject,
-                To = new List<EmailAddress>
-                {
-                    new EmailAddress(Empty, expectedContents.To)
-                }
             };
-            
+
             var actual = (await _emailServerDriver.FindAllEmailsAsync()).First();
 
             actual.Should().BeEquivalentTo(expected, conf => conf.IncludingAllDeclaredProperties());
         }
 
+        [Then(@"the patient numbers price type attachment contains the correct information")]
+        public async Task ThenThePatientNumbersPriceTypeAttachmentContainsTheCorrectInformation()
```

Comment:
The context of the step is very specific to the Update order status feature. Do you think it would be better if this step was placed in the `OrderStatusSteps` class? I believe this way you could use the details of the request and not have to store the orderid within the scenario context.